doi/build.gradle

```diff
@@ -33,7 +33,9 @@ dependencies {
     implementation 'org.opencadc:cadc-access-control-identity:[1.1.0,)'
     implementation 'org.opencadc:cadc-vosi:[1.4.3,2.0)'
     implementation 'org.opencadc:cadc-vos:[2.0.7,3.0)'
+    implementation 'org.opencadc:cadc-uws:[1.0.5,)'
```

Comment:
comments on the build:

The cadc-uws requirement is new/missing because cadc-vos used to pull it in; the real "bug" is that cadc-vos declares it as an implementation dependency when in fact it exposes cadc-uws classes in it's API, so it should declare it as an `api` dependency. So this may really a bug in the cadc-vos/build.gradle... actually, cadc-vos-client does declare cadc-uws as an api dependency so this should not be required to build. But I also see that cadc-vos and cadc-vos-client in vos.git main branch are ahead of maven central so I think I fixed this but never published it. I will do that asap.

@jburke-cadc made a change (just merged into main) that removes the need for commons-io so that can be removed